homeassistant/components/sensor/miflora.py

```diff
@@ -31,9 +29,7 @@
 DEFAULT_NAME = 'Mi Flora'
 DEFAULT_RETRIES = 2
 DEFAULT_TIMEOUT = 10
-
-UPDATE_INTERVAL = 1200
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=UPDATE_INTERVAL)
```

Comment:
If you rename this to SCAN_INTERVAL, Home Assistant will only call `update` every 1200 seconds.